user-interface/src/main/java/life/qbic/datamanager/views/login/passwordreset/ResetPasswordComponent.java

```diff
@@ -0,0 +1,128 @@
+package life.qbic.datamanager.views.login.passwordreset;
+
+import com.vaadin.flow.component.ComponentEvent;
+import com.vaadin.flow.component.ComponentEventListener;
+import com.vaadin.flow.component.Key;
+import com.vaadin.flow.component.Text;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.H2;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.textfield.TextField;
+import com.vaadin.flow.data.binder.Binder;
+import com.vaadin.flow.data.binder.ValidationResult;
+import com.vaadin.flow.data.binder.Validator;
+import com.vaadin.flow.data.validator.EmailValidator;
+import com.vaadin.flow.router.RouterLink;
+import com.vaadin.flow.spring.annotation.SpringComponent;
+import com.vaadin.flow.spring.annotation.UIScope;
+import java.io.Serial;
+import java.util.Objects;
+import life.qbic.datamanager.views.general.CardLayout;
+import life.qbic.datamanager.views.notifications.ErrorMessage;
+import life.qbic.datamanager.views.register.UserRegistrationMain;
+import life.qbic.identity.api.UserInformationService;
+
+/**
+ * Reset Password Component
+ * <p>
+ * Card Stylized component similar to {@link com.vaadin.flow.component.login.LoginOverlay}
+ * component, Providing the input fields necessary to enable a user to specify his email for which
+ * the password should be reset
+ */
+@SpringComponent
+@UIScope
+public class ResetPasswordComponent extends CardLayout {
```

Comment:
According to the javadoc this component and NewPasswordSetComponent have the same purpose?

Reply:
Nice catch 👁️. I updated the JD to better reflect their purpose.
(Reset Password --> Inform user about new password set)